File: server/EzyRoulette-app-api/src/main/java/org/youngmonkeys/app/entity/Bettor.java
package org.youngmonkeys.app.entity;

import com.tvd12.ezyfox.binding.annotation.EzyObjectBinding;
import com.tvd12.ezyfox.database.annotation.EzyCollection;
import lombok.Data;

@Data
@EzyObjectBinding
public class Bettor implements Comparable<Bettor>{
    private String username;
    private float betAmount;

    public Bettor(String username, float betAmount) {
        this.username = username;
        this.betAmount = betAmount;
    }

    @Override
    public String toString() {
        return username +"\t"+ betAmount;
    }

    @Override
    public int compareTo(Bettor b) {
        if(getBetAmount()<=b.getBetAmount()) return 1;
        return -1;
    }
}

File: unity-client/Assets/2 - Scripts/entities/Bettor.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

public class Bettor 
{
    private string username;
    private float betAmount;

    public Bettor(string username, float betAmount)
    {
        this.username = username;
        this.betAmount = betAmount;
    }

    public string Username { get => username; set => username = value; }
    public float BetAmount { get => betAmount; set => betAmount = value; }
    
}
